src/components/Questionnaire/QuestionTypes/DiagnosisQuestion.tsx

```
@@ -128,6 +129,8 @@
 }: DiagnosisQuestionProps) {
   const { t } = useTranslation();
 
+  const { t } = useTranslation();
```

Biome (1.9.4)

[error] Shouldn't redeclare 't'. Consider to delete it or rename it.

't' is defined here:


(lint/suspicious/noRedeclare)